docs/installation.rst

```diff
@@ -390,9 +390,10 @@ After navigating to the Pillow directory, run::
 Build Options
 """""""""""""
 
-* Environment variable: ``MAX_CONCURRENCY=n``. Pillow can use
-  multiprocessing to build the extension. Setting ``MAX_CONCURRENCY``
-  sets the number of CPUs to use, or can disable parallel building by
+* Config setting: ``-C parallel=n``. Can also be given
```

Comment:
Why not use the same naming pattern for settings and env vars?

Reply:
`MAX_CONCURRENCY` was added in #721 (almost 10 years ago).

`--parallel` is a distutils/setuptools flag added in Python 3.5; Pillow support added in https://github.com/python-pillow/Pillow/commit/64bce1a583ffdbf1811263ae7dce3088a669d7c7 (over 5 years ago).

So I think it makes sense to keep those names for consistency.